backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/exceptions/AnnouncementExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.exceptions;
+
+import de.bund.digitalservice.ris.norms.application.exception.AnnouncementNotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * Custom exception handler for handling exceptions that are related to an Announcement. This class
+ * is annotated with {@link ControllerAdvice} to allow centralized exception handling for all
+ * controllers in the application.
+ */
+@Order(1)
+@ControllerAdvice
+@Slf4j
+public class AnnouncementExceptionHandler {
+
+  private static final String CONTENT_FORMAT_TEMPLATE = "{\"message\": \"%s\"}";
+
+  /**
+   * Exception handler method for handling {@link AnnouncementNotFoundException}.
+   *
+   * @param e The exception that occurred.
+   * @return A {@link ResponseEntity} with an HTTP 404 status and the exception message.
+   */
+  @ExceptionHandler(AnnouncementNotFoundException.class)
+  @ResponseStatus(HttpStatus.NOT_FOUND)
+  public ResponseEntity<String> handleException(final AnnouncementNotFoundException e) {
```

Comment:
Todo: can you please move this to `NormExceptionHandler`? That one is meant to handle our custom exceptions, so we don't need to create a new ControllerAdvice for every single exception we have.

(as a side note, I'll admit that the name `NormExceptionHandler` is a bit confusing. "Norm" in this context doesn't mean our Norm domain entity but "the Norms application")

Reply:
Right, it was the naming that made me create a new one. Will move.